content/sensu-go/5.12/installation/auth.md

```diff
@@ -1094,6 +1098,170 @@ example      | {{< highlight shell >}}"name": "activedirectory"{{< /highlight >}
+
+The Sensu offers license-activated support for OIDC driver for using the OpenID Connect 1.0 protocol (OIDC) on top of the OAuth 2.0 protocol for RBAC authentication.
+
+### OIDC configuration examples
+
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+---
+type: oidc
+api_version: authentication/v2
+spec:
+  additional_scopes:
+  - groups
+  - email
+  client_id: a8e43af034e7f2608780
+  client_secret: b63968394be6ed2edb61c93847ee792f31bf6216
+  redirect_uri: http://127.0.0.1:8080/api/enterprise/authenication/v2/oidc/callback
+  server: https://localhost:9031
+  groups_claim: groups
+  groups_prefix: 'oidc:'
+  username_claim: email
+  username_prefix: 'oidc:'
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+   "type": "oidc",
+   "api_version": "authentication/v2",
+   "spec": {
+      "additional_scopes": [
+         "groups",
+         "email"
+      ],
+      "client_id": "a8e43af034e7f2608780",
+      "client_secret": "b63968394be6ed2edb61c93847ee792f31bf6216",
+      " redirect_uri": "http://127.0.0.1:8080/api/enterprise/authenication/v2/oidc/callback",
+      "server": "https://localhost:9031",
+      "groups_claim": "groups",
+      "groups_prefix": "oidc:",
+      "username_claim": "email",
+      "username_prefix": "oidc:"
+   }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+### RBAC for OIDC definition specification
+
+#### `oidc` attributes
+
+| client_id    |      |
+-------------|------
+description  | The OIDC provider application "Client ID" _NOTE: requires [registration of an application in the OIDC provider][27]._
+required     | true
+type         | String
+example      | {{< highlight shell >}}"client_id": "1c9ae3e6f3cc79c9f1786fcb22692d1f"{{< /highlight >}}
+
+| client_secret  |      |
+-------------|------
+description  | The OIDC provider application "Client Secret" _NOTE: requires [registration of an application in the OIDC provider][27]._
+required     | true
+type         | String
+example      | {{< highlight shell >}}"client_secret": "a0f2a3c1dcd5b1cac71bf0c03f2ff1bd"{{< /highlight >}}
+
+| server |  |
+-------------|------
+description  | The location of the OIDC server you wish to authenticate against. _NOTE: Configuring with http will cause the connection to  be insecure._
+required     | true
+type         | String
+example      | {{< highlight shell >}}"name_attribute": "https://sensu.oidc.provider.example.com"{{< /highlight >}}
+
+| redirect_uri |   |
+-------------|------
+description  | Redirect URL to provide to the OIDC provider. Requires `/api/enterprise/authenication/v2/oidc/callback` _NOTE: only required for certain OIDC providers, such as Okta._
+required     | false
+type         | String
+example      | {{< highlight shell >}}"redirectURL": "http://localhost:8080/api/enterprise/authenication/v2/oidc/callback"{{< /highlight >}}
+
+| groups_claim |   |
+-------------|------
+description  | The claim to use to form the associated RBAC groups. 
+required     | false
+type         | Array
```

Comment:
Testing this for sensu-puppet module using CI builds and seems `groups_claim` is a String.